src/vm.c

```diff
@@ -275,9 +285,58 @@ int vm_irq_line(vm_t *v, int irq, int level)
     return 0;
 }
 
+int vm_irq_trigger(vm_t *v, int irq)
```

Comment:
When will this function be used?

Reply:
It is a replacement of irqfd and can be removed.